GateWatcher/README.md

```diff
@@ -0,0 +1,7 @@
+# GateWatcher
```

Comment:
The logo file should be:
- max 50 KB (+57.6 KB for this one)
- transparent (I don't think that this one is)